### DISCUSSION TITLE
Best SD Card models?

### DISCUSSION THREAD

**Discussion by chrbar11** [roleoroleo/yi-hack-Allwinner-v2#956] [General]
Hi,

What SD Card type, size and brand do you recommend?

I've Yi 1080p Home and Yi Outdoor 1080p cameras.
Yi recommend 8 to 64GB Micro SD card, Class 10, FAT32 formatted.
https://help.yitechnology.com/hc/en-us/articles/360053511593-What-is-the-recommended-microSD-card-for-YI-cameras

I plan to order SanDisk Extreme PRO microSDXC UHS-I CARD 64GB
https://shop.sandisk.com/products/memory-cards/microsd-cards/sandisk-extreme-pro-uhs-i-microsd?sku=SDSQXCU-064G-GN6MA

Thank you a lot for your help and advices.
Chris

**Comment by roleoroleo** [comment]
Use a good quality SD card: samsung, sandisk for example.
Don't use chinese sd card.
About the size, it depends on the usage you want to do. 16 GB is enough, I think.
Finally, about the speed, I think it's not important, the bus of this cheap hardware is not fast. So, a fast SD card is not really used.
